conferences/2025-agu.md

```diff
@@ -37,3 +37,42 @@ We're excited to hear what you have to say at our session!
 ## Invited speakers
 
 _TBD_
+
+
+## Organizers
+
+
+### Primary Convener: Qiusheng Wu
+
+_TODO: Insert mini-bio_
```

Comment:
@giswqs please suggest a bio! Can you please also provide a link or paste a photo into the comments here?

Reply:
Re-pasting to upload to user-attachments so I can link to it without copying the image into the repo :)  
<img width="400" height="400" alt="image" src="https://github.com/user-attachments/assets/abb6fb5f-9307-4ba8-862c-6ecdab2240a9" />

Reply:
Thank you!